spec/requests/api/v1/events/nearest_spec.rb

```diff
@@ -0,0 +1,64 @@
+describe 'Events Requests' do
+  describe 'GET /v1/events/nearests?lat=&lon=&radius=' do
+    it 'returns events closest to lat and lon' do
+      near_event = create(:event,
+                          lat: 37.760322,
+                          lon: -122.429667)
+      farther_event = create(:event,
```

Comment:
You might make this more dramatically farther. You can get lat/lon from from google maps by right-clicking and looking in the upper left corner under the search box.
![screen shot 2015-01-12 at 2 10 52 pm](https://cloud.githubusercontent.com/assets/49549/5709102/ecdb2578-9a64-11e4-8a81-ce2b869c97b4.png)